README.md

```diff
@@ -1,89 +1,99 @@
-<br><br><br>
+<a name="readme-top"></a>
 
-# To-Do List
+<!-- TABLE OF CONTENTS -->
 
-to-do list version 1.0.0
-> list structure
+# 📗 Table of Contents
 
-<br>
-<br>
+- [📖 About the Project](#about-project)
+  - [🛠 Built With](#built-with)
+  - [🚀 Live Demo](#live-demo)
+- [💻 Getting Started](#getting-started)
+- [👥 Authors](#authors)
+- [🤝 Contributing](#contributing)
+- [⭐️ Show your support](#support)
```

Comment:
- [ ] I am writing this here since you have made no changes on the other files.
You need to have an array of some simple to-do tasks (array of objects) with each object containing three keys - description [string], completed [bool], index: [number]. Also remember that on page load, the task array is rendered dynamically to the DOM. The tasks should appear in order of the `index` values for each task.

Reply:
- [ ] Style your app according to this given [video](https://www.youtube.com/watch?v=AcUd-_Yjjqg).[captured video](https://www.youtube.com/watch?v=AcUd-_Yjjqg).

Reply:
- [ ] You have some broken images on your buttons, kindly add remove them because they do not look broken 
![image](https://user-images.githubusercontent.com/62705947/206218778-0be16cf6-42db-4d70-9439-2f54a923660d.png)

Reply:
Thanks for the head's up! I think you'll find everything in order now 👍